deploy/demo.sh

```diff
@@ -13,6 +13,7 @@ kubectl="${HOME}/.kubeadm-dind-cluster/kubectl"
 BASE_LOCATION="${BASE_LOCATION:-https://raw.githubusercontent.com/Mirantis/virtlet/master/}"
 # Convenience setting for local testing:
 # BASE_LOCATION="${HOME}/work/kubernetes/src/github.com/Mirantis/virtlet"
+DEPLOY_LOG_CONTAINER=${DEPLOY_LOG_CONTAINER:-0} # 0 = don't deploy, 1 = deploy, 2 = inject local log image and deploy
```

Comment:
Would you bi willing to consider changing 0, 1, 2 into meaningful options? For example 

* empty (undefined) == 0
* deploy == 1
* inject == 2

Then you simply change this to `DEPLOY_LOG_CONTAINER=${DEPLOY_LOG_CONTAINER:-}`. I think it will be easier for the users to comprehend when they need to set this env var to `deploy` or `inject` than `1` or `2`.